docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,84 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's typically only one @Mod in your mod as a whole, sitting on top of your entry class.
+
+Components:
+
+`modid` [Required]
+This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only used internally, but can be seen by users who dig around. Other mods also require this ID if they want to find each other (APIs for example).
+If you want to know more about annotations in general you can check out [Oracle's docs](https://docs.oracle.com/javase/tutorial/java/annotations/index.html) on it.
+
+Example: `@Mod(modid="examplemod")`
```

Comment:
Is there any damage or overall decrease of quality of the doc to include these examples?